benchopt/tests/test_benchmarks.py

```diff
@@ -114,6 +115,19 @@ def test_solver_install_api(benchmark, solver_class):
 
 @pytest.mark.requires_install
 def test_solver_install(check_test, test_env_name, benchmark, solver_class):
+    # Skipping the specific R-PGD solver for Windows
+    if sys.platform == 'win32':
+        if hasattr(benchmark, 'name'):
+            benchmark_name = benchmark.name
+        else:
+            benchmark_name = str(benchmark)
+        solver_class_name = solver_class.__module__ + "."
+        solver_class_name += solver_class.__name__
+        if (
+            benchmark_name == "dummy_benchmark" and solver_class_name ==
+            "benchopt_benchmarks.dummy_benchmark.solvers.r_pgd.Solver"
+        ):
+            pytest.skip("Skipping the specific R-PGD solver for Windows")
```

Comment:
It would be better if we can set this skip directly in the benchmark: https://github.com/benchopt/benchopt/blob/main/benchopt/tests/dummy_benchmark/test_config.py